tests/bluetooth/tester/src/btp_l2cap.c

```diff
@@ -19,14 +19,15 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
 
 #include "btp/btp.h"
 
-#define DATA_MTU_INITIAL 128
-#define DATA_MTU 256
-#define DATA_BUF_SIZE BT_L2CAP_SDU_BUF_SIZE(DATA_MTU)
+#define L2CAP_MPS 96
```

Comment:
Consider setting this to `BT_L2CAP_RX_MTU`, if this is an arbitrary value. `BT_L2CAP_RX_MTU` is the maximum supported by the Host.

Reply:
yeap,

although this is to match current PTS workspace configuration in autopts, we can change it later on but I didn't wanted to touch workspace yet

Reply:
@sjanc Should we consider have a ixit.h file that contains all the configured values for PTS? Then it will be easier to determine if a value is a specific value due to IXIT, or just some arbitrary value, and whether modifying said value requires a change in IXIT as well?